.gitignore

```diff
@@ -10,4 +10,6 @@
 .vscode
 
 # macOS
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+vendor/
```

Comment:
Hi, what does this do? Is there anything that writes to `vendor/`?

Reply:
If I download dependencies from `go mod vendor` then it getting tracked by git.